### ISSUE TITLE
Saving workspace with references to objects in the world creates errors

### ISSUE THREAD

**Issue by Paula-Kli** [LivelyKernel/lively.next#239]
When you open a workspace and write something like:
```
import { Morph } from "lively.morphic";
import { pt } from "lively.graphics";
let rec = new Morph({extent: pt(100,100)})
rec.openInWorld()
```
Saving the world after opening the Morph creates following error message with the StatusMessages:
```
Error when trying to update AttributeConnection(<StatusMessageForMorph - messageMorph>.extent --> <StatusMessageForMorph - messageMorph>.relayout) with value pt(400.0,35.0):
TypeError: Cannot read property 'length' of undefined
TypeError: Cannot read property 'length' of undefined
```
The world is saveable and loadable. Nevertheless the reference to the morph from the workspace is lost (which might not be a problem)

**Comment by merryman** [comment]
I could not reproduce this on the latest stage of master. Can you try updating to the latest version?

**Comment by Paula-Kli** [comment]
Maybe this issue is something locally but I am on the latest state of the master and it is a new world where I did not do anything yet except what you can see in the screenshot

![screenshot_saving](https://user-images.githubusercontent.com/49535291/114307873-e7ab1480-9ae1-11eb-9775-473a04baff09.png)

**Comment by T4rikA** [comment]
I could also reproduce this error.

**Comment by frcroth** [comment]
Can reproduce this

**Comment by merryman** [comment]
Fixed in: 82f80182.
